onnx/onnx_pb.h

```diff
@@ -7,7 +7,7 @@
 #ifdef ONNX_ML
 #include "onnx/onnx-ml.pb.h"
 #else
-#include "onnx/onnx.pb.h"
+#include "onnx.pb.h"
```

Comment:
Why changing this?

Reply:
Aha, it's because that the header generated by protoc is put in /build folder directly. Yanqing asked the same question.